strings/remove_vowels.py

```diff
@@ -0,0 +1,37 @@
+# This function removes all the vowels present in the given sentence or a word.
+# returns the modified version of that string.
+
+
+def remove_vowels(sentence: str) -> str:
+    """
+    This remove_vowels function removes all the vowels
+    from a given sentence or a word
+    >>> remove_vowels('Hello world')
+    'Hll wrld'
+    >>> remove_vowels('Programming is my passion')
+    'Prgrmmng s my pssn'
+    >>> remove_vowels('I love coding 24 hours')
+    ' lv cdng 24 hrs'
+    >>> remove_vowels('')
+    ''
+    """
+
+    # List of all the vowels
+    vowel_list = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
+
+    # iterating over a list of vowels
+    for vowel in vowel_list:
+        # replacing the vowel with blank
+        sentence = sentence.replace(vowel, "")
+
+    # returning a updated string
+    return sentence  # return --str
```

Comment:
```suggestion
    return sentence
```
You already specify at the function declaration that the return type is str